terra-jupyter-aou/Dockerfile

```diff
@@ -41,6 +41,7 @@ ENV PIP_USER=false
 # this is effectively ignored. On GCE VMs, this will result in failures to
 # import the pyspark package.
 ENV PYTHONPATH $PYTHONPATH:/usr/lib/spark/python
+ENV PYSPARK_PYTHON=python3
```

Comment:
@calbach FYI - setting this EV in the aou and hail images, and I removed the custom kernelspec in the hail image.

Reply:
Thanks!